js/classes/player.js

```diff
@@ -1,13 +1,39 @@
 class Player extends Game {
```

Comment:
Your player class shouldn't be extending your Game class -- this means that your players are inheriting all the instance properties and methods of Game, which they don't need in order to function.